src/OpenTelemetry.Instrumentation.Hangfire/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+* Downgrade `OpenTelemetry.Api.ProviderBuilderExtensions` to `1.6.0` to avoid
+  `Microsoft.Extensions.*` v8 dependencies.
+  ([#1502](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1502))
+
```

Comment:
It is in conflict with the next statement.
What is more, it is not true in context of previous release.

Can it be something like

```
* Update `OpenTelemetry.Api.ProviderBuilderExtensions` to `1.6.0`.
  * Update `OpenTelemetry.Api` to `1.6.0`.
  ([#1486](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1502))
```
?

Reply:
Maybe I just need to put it in as a last note rather than the first. So it will say `Update` then `Downgrade`. It can't be update to 1.7 and then update to 1.6.

Reply:
In the end, we finish with 1.6. So it must be `downgrade`.

Reply:
Upgrade, the previous release was on 1.5.0 : )

Reply:
So, should I remove the mention of 1.7.0 from the changelog?

Reply:
It confusing to see update to 1.6 and update to 1.7 in the same release, and find out the final version is 1.6.

Reply:
You could maybe strikeout the 1.7.0 ?

Reply:
IMO, this will be a little bit weird. Maybe something like:
```
* Revert the update `OpenTelemetry.Api.ProviderBuilderExtensions` to `1.7.0` and update to `1.6.0` instead.
  * Revert the update `OpenTelemetry.Api` to `1.7.0` and update to `1.6.0` instead.
  ([#1486](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1502))
```

Reply:
Consider the whole section

```md
* Update `OpenTelemetry.Api.ProviderBuilderExtensions` to `1.6.0`.
  * Update `OpenTelemetry.Api` to `1.6.0`.
  ([#1344](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1344))
* Added overloads which accept a name to the `TracerProviderBuilder`
  `HangfireInstrumentationOptions` extension to allow for more fine-grained
  options management
  ([#1442](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1442))
* Add Filter to HangfireInstrumentationOptions.
  ([#1440](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1440))
```